hbase-client/src/main/java/org/apache/hadoop/hbase/filter/PrefixFilter.java

```diff
@@ -142,6 +165,28 @@ boolean areSerializedFieldsEqual(Filter o) {
     return Bytes.equals(this.getPrefix(), other.getPrefix());
   }
 
+  @Override
```

Comment:
Let's move these before the serialization-realted methods, so that the real logic is together,

Reply:
Thanks, good idea.
I moved them.